src/components/Options/index.tsx

```diff
@@ -107,25 +112,27 @@ export default class Options extends Component<Props, State> {
 
     return (
       <div class={`${style.options}${className ? (' ' + className) : ''}`}>
-        <p>Quantization</p>
-        <label>
-          <input
-            name="quantizer.enable"
-            type="checkbox"
-            checked={!!preprocessorState.quantizer.enabled}
-            onChange={this.onPreprocessorEnabledChange}
-          />
-          Enable
-        </label>
-        {preprocessorState.quantizer.enabled ? (
-          <QuantizerOptionsComponent
-            options={preprocessorState.quantizer}
-            onChange={this.onQuantizerOptionsChange}
-          />
-        ) : (
-          <div/>
+        {encoderState.type !== 'identity' && (
```

Comment:
Not saying it needs to be in this PR, but do you think it'd be worth having a property on each encoder indicating whether we should show/allow quantization as a preprocessing step?

Reply:
When wouldn't we allow it?